src/index.js

```diff
@@ -1,19 +1,12 @@
-// eslint-disable-next-line no-unused-vars
+/* eslint-disable no-unused-vars */
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-// eslint-disable-next-line no-unused-vars
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
- [ ] All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. Please, let's delete `logo192.png`, `logo512.png`,  and `favicon.ico`